grammar.y

```diff
@@ -1347,6 +1436,9 @@ paren_update_list:
     } else {
       exprs := make([]*UpdateExpr, len($2))
       for i := 0; i < len($2); i++ {
+        if isRowID($2[i].Name) {
```

Comment:
Block references to `ROWID` on an `UPDATE` (this is the parenthesise `UPDATE` syntax; e.g. `update t set (rowid) = (1);`)